node/src/run_modes_factories.rs

```diff
@@ -61,11 +61,7 @@ pub trait ServerInitializerFactory {
     fn make(&self) -> Box<dyn ServerInitializer<Item = (), Error = ()>>;
 }
 pub trait DaemonInitializerFactory {
-    fn make(
-        &self,
-        args: &[String],
-        streams: &mut StdStreams,
-    ) -> Result<Box<dyn DaemonInitializer>, ConfiguratorError>;
+    fn make(&self, args: &[String]) -> Result<Box<dyn DaemonInitializer>, ConfiguratorError>;
```

Comment:
are these code changes just cleaning up formatting etc?

Reply:
These formatting changes we recommended by IntelliJ.